docs/source/tutorial/run_project.rst

```diff
@@ -48,9 +48,9 @@ Sometime you will need *all* the simulation results including timeseries results
 
 .. _download:
    
-.. todo::
-   
-   Only 150 datapoints are downloading right now. There's a ticket to fix that. Ry has a script to do the downloading of all datapoints. Add those instructions here.
+.. note::
+
+   From the PAT interface, only 150 datapoints are downloading right now. For downloading all datapoints, run the ``scripts/download_datapoints.rb`` script. The script requires that the ``localResults/results.csv`` be downloaded first. Supply all required arguments to the script: project directory, server DNS, and analysis ID.
```

Comment:
Can you include an example command line input to call the script? e.g., `ruby download_datapoints.rb --dir=<project/directory> --server=<server> --analysisID=<ID>` or whatever it would be?